tests/publiccloud/transfer_repos.pm

```diff
@@ -25,8 +25,8 @@ sub run {
     my $remote = $args->{my_instance}->username . '@' . $args->{my_instance}->public_ip;
     my @addons = split(/,/, get_var('SCC_ADDONS', ''));
     my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', 0);
-    # /opt folder in Hardened images requires extra permissions so we will use /tmp instead
-    my $repodir = is_hardened() ? "/tmp/repos/" : "/opt/repos/";
+    # $repodir folder in Hardened images requires extra permissions, set by chmod
```

Comment:
Let's remove this comment.

Reply:
Thinking... it could address future questions, like last issues... but removing it code is much clean